Journalator/Source/SlashCmd.lua

```
@@ -56,13 +52,103 @@ function Journalator.SlashCmd.Debug(...)
+      Journalator.Utilities.Message("Failed to create portfolio. Check your input.")
+    end
+  elseif command == "add" then
+    local portfolioId = tonumber(select(2, ...))
+    local itemName = select(3, ...)
+    local amount = select(4, ...)
+    if not portfolioId or not itemName or not amount then
+      Journalator.Utilities.Message("Usage: /jnr invest add <portfolioId> <itemName> <amount>")
+      Journalator.Utilities.Message("Example: /jnr invest add 1 \"Iron Ore\" 100000")
+      return
+    end
+    local amountValue = Journalator.InvestONator.ParseGoldInput(amount)
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'Journalator'

(W113)